lib/config/fileconf.go

```diff
@@ -609,6 +615,32 @@ func checkAndSetDefaultsForAzureMatchers(matcherInput []AzureMatcher) error {
 	return nil
 }
 
+func checkAndSetDefaultsForAzureInstaller(matcher *AzureMatcher) error {
+	if matcher.InstallParams == nil {
+		matcher.InstallParams = &InstallParams{
+			JoinParams: JoinParams{
+				TokenName: defaults.AzureInviteTokenName,
+				Method:    types.JoinMethodAzure,
+			},
+			ScriptName: installers.InstallerScriptName,
+		}
+	} else {
```

Comment:
```suggestion
		}
		return nil
	}

```

Same spirit as https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow, but this is obviously not an error flow, just a guard/special case.